src/main/java/appeng/recipes/game/WirelessTerminalRecipe.java

```diff
@@ -0,0 +1,145 @@
+package appeng.recipes.game;
+
+import appeng.api.AEApi;
+import appeng.api.definitions.IComparableDefinition;
+import appeng.api.definitions.IDefinitions;
+import net.minecraft.init.Items;
+import net.minecraft.inventory.IInventory;
+import net.minecraft.inventory.InventoryCrafting;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.IRecipe;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.world.World;
+import net.minecraftforge.oredict.OreDictionary;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+
+/**
+ * Created by Tom on 10/2/2015.
+ */
+public class WirelessTerminalRecipe implements IRecipe {
+
+    private final IComparableDefinition terminal;
+    private final ItemStack waterBucket;
+
+    private static final ArrayList<String> dyes = new ArrayList<String>();
+    static {
+        dyes.add( "dyeBlack" );
+        dyes.add( "dyeBlue" );
+        dyes.add( "dyeBrown" );
+        dyes.add( "dyeCyan" );
+        dyes.add( "dyeGray" );
+        dyes.add( "dyeGreen" );
+        dyes.add( "dyeLightBlue" );
+        dyes.add( "dyeLightGray" );
+        dyes.add( "dyeLime" );
+        dyes.add( "dyeMagenta" );
+        dyes.add( "dyeOrange" );
+        dyes.add( "dyePink" );
+        dyes.add( "dyePurple" );
+        dyes.add( "dyeRed" );
+        dyes.add( "dyeWhite" );
+        dyes.add( "dyeYellow" );
+    }
+
+    public WirelessTerminalRecipe()
+    {
+        final IDefinitions definitions = AEApi.instance().definitions();
+        terminal = definitions.items().wirelessTerminal();
+        waterBucket = new ItemStack( Items.water_bucket, 1 );
+    }
+
+    @Override
+    public boolean matches( InventoryCrafting inv, World w ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)